include/splinterdb/splinterdb.h

```diff
@@ -32,12 +53,18 @@ typedef struct {
    // For a simple reference implementation, see default_data_config.h
    data_config *data_cfg;
 
-
    // optional advanced config below
    // if unset, defaults will be used
    void *heap_handle;
    void *heap_id;
 
+   // Shared memory support
+   uint64 shmem_size;
```

Comment:
Few externally visible config options. Intended for internal use now. See big block comment above.